Hooks/BeforeScenario.cs

```diff
@@ -0,0 +1,37 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using ProjectMarsAutomation.Pages;
+using ProjectMarsAutomation.utitlities;
+
+namespace ProjectMarsAutomation.Hooks
+{
+    [Binding]
+    public sealed class BeforeScenario : CommonDriver
```

Comment:
rename BeforeScenario to Hooks
and then dont extend class to CommonDriver  e.g -> 
( :CommonDriver)